core/src/main/scala/io/finch/internal/ToResponse.scala

```diff
@@ -56,6 +58,21 @@ object ToResponse extends LowPriorityToResponseInstances {
     rep
   }
 
+  implicit def asyncStreamToResponse[A, CT <: String](implicit
```

Comment:
It could be nice to use new-line delimeter, but it works correctly in terms of _streaming_